packages/account/src/Components/poi/status/unsupported/unsupported.jsx

```diff
@@ -19,12 +20,18 @@ const checkNimcStep = documents => {
 
 const Unsupported = ({ country_code, ...props }) => {
     const [detail, setDetail] = React.useState(null);
-    const toggleDetail = index => setDetail(index);
+    const toggleDetail = index => {
+        setDetail(index);
+    };
     const documents = getDocumentIndex({
         setDetail,
         country_code,
     });
 
+    if (props?.manual && props.manual.status === 'pending') {
```

Comment:
Can't be like this?
```suggestion
    if (props?.manual?.status === 'pending') {
```

Reply:
It can be as you suggested. I think splitting makes the code more readable

Reply:
@likhith-deriv But I think using an optional chain is readable, too.